service/src/main/java/de/adorsys/psd2/sandbox/xs2a/service/ais/AccountSpiImpl.java

```diff
@@ -224,4 +225,8 @@ public AccountSpiImpl(TestDataService testDataService, TestDataMapper testDataMa
     return testDataService
         .getTransactions(psu.get().getPsuId(), accountId);
   }
+
+  private boolean dateIsInDateRange(LocalDate date, LocalDate from, LocalDate to) {
+    return date.isAfter(from) && date.isBefore(to);
```

Comment:
What about the edge cases? If `date == to` or `date == from` date should be included, right?

Reply:
Done